src/components/Contentful/InlineImages.js

```diff
@@ -23,13 +23,16 @@ export default function ContentfulInlineImages({
 
   let captionComponent
 
-  if (caption) captionComponent = <p className="inline-images__caption">{caption}</p>
+  if (caption)
+    captionComponent = <p className="inline-images__caption">{caption}</p>
```

Comment:
Could we have some curly braces around the body of this if statement? These make me a little nervous (personal preference I realise)

Reply:
I didn't actually add this, we just have an autoformatter on when files are saved now. I'd prefer not to touch it.

Reply:
Ah ok, fair enough